packages/sdk/.eslintrc.cjs

```diff
@@ -1,3 +1,12 @@
 module.exports = {
   extends: '../../.eslintrc.cjs',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    // ecmaVersion: 12,
```

Comment:
I was experiencing issues with the linter by myself in my last PR. I have used this settings and works pretty fine. Why the commented code? If we are not using it, we should remove it

Reply:
It will be rebased against deposit's PRs